linkerd/cache/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use std::fmt;
+
+#[derive(Debug)]
+pub struct NoCapacity(pub usize);
+
+impl fmt::Display for NoCapacity {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "router capacity reached ({})", self.0)
```

Comment:
nit: should this now be
```suggestion
        write!(f, "cache capacity reached ({})", self.0)
```